src/dapla_pseudo/v1/models.py

```diff
@@ -136,16 +87,122 @@ def keyset_list(self) -> t.Union[t.List[PseudoKeyset], None]:
         return None if self.keyset is None else [self.keyset]
 
 
+class PseudoFunctionArgs(BaseModel):
+    """Representation of the possible keyword arguments"""
+
+    def __str__(self) -> str:
+        """As a default, represent the fields of the subclasses as kwargs on the format 'k=v'."""
+        return ",".join(f"{k}={v}" for k, v in self.model_dump(by_alias=True).items() if v is not None)
+
+    model_config = ConfigDict(alias_generator=camelize, populate_by_name=True)
+
+
+class MapSidKeywordArgs(PseudoFunctionArgs):
+    """Representation of kwargs for the 'map-sid' function."""
+
+    key_id: PredefinedKeys | str = PredefinedKeys.PAPIS_COMMON_KEY_1
+    version_timestamp: t.Optional[str] = None
```

Comment:
Do we know anything about what format these *should* take? Ideally we would validate the format here before we send the Pseudo Request. But possibly it's not well enough defined by FREG?